src/init.cpp

```diff
@@ -528,6 +529,14 @@ static void BlockNotifyCallback(const uint256& hashNewTip)
     boost::thread t(runCommand, strCmd); // thread runs free
 }
 
+static void TrxExpiryNotifyCallback(const std::string& txid)
```

Comment:
Consistent abbreviation:

```suggestion
static void TxExpiryNotifyCallback(const std::string& txid)
```